_definitions/sandor-clegane.mdd

```diff
@@ -1,7 +1,6 @@
 ---
 title: Sandor Clegane
 terms: [Sandor]
-house: Clegane
```

Comment:
Honestly can't remember why I added this.

Have found one reference in code (https://github.com/wjdp/gotdict/blob/cdc84de9c2e0182466a24fe56ba77edc2e23df68/_plugins/indexer.rb#L17) but happy to remove from defs as it doesn't seem to do anything.